### ISSUE TITLE
downloading voices became harder

### ISSUE THREAD

**Issue by denizsincar29** [mush42/piper-nvda#11]
Hello. I tried to download voices for piper nvda but failed. there are now no tar archives in the download page, but rather onnx and json files in the huggingface website that the voice samples redirect to. It is not hard to me, but can be hard to people who cannot copy that files into nvda folder

**Comment by mush42** [comment]
Hi @denizsincar29 

I understand your concern, but we can do nothing about this unless we host the voices ourselves.

If we have the means to host those voices in our own server, then we can collect official and community trained voices, and make downloading and installing them much easier via an integrated voice gallery in the add-on itself.

Best
Musharraf

**Comment by vortex1024** [comment]
@mush42 , I can provide hosting if you are interested.

**Comment by zersiax** [comment]
@denizsincar29  what is the procedure now? do you need the .json and .onnx files, and where are they pasted?

**Comment by vortex1024** [comment]
the addon now has a voice manager which handles downloading and installing automatically, but you are going to have to copy the files from the code, as a new release has not been made.
